frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -449,7 +454,9 @@ export default ({
       } else {
         events = await sbp('chelonia/out/eventsBefore', before, limit)
       }
-
+      if (curChatRoomId !== this.currentChatRoomId) {
+        return
+      }
```

Comment:
This needs a comment above it explaining why this is here

Reply:
Got it.